Common/include/mbedtls_error_utils.h

```diff
@@ -47,22 +47,22 @@
     mbedtls_low_level_strerr( mbedTlsCode ) : ( const char * ) "<No-Low-Level-Code>"
 #endif /* mbedtlsLowLevelCodeOrDefault */
 
-#define MBEDTLS_MSG_IF_ERROR( lError, pMessage )            \
-    do                                                      \
-    {                                                       \
-        if( lError < 0 )                                    \
-        LogError( pMessage " %s : %s.",                     \
-                  mbedtlsHighLevelCodeOrDefault( lError ),  \
-                  mbedtlsLowLevelCodeOrDefault( lError ) ); \
+#define MBEDTLS_MSG_IF_ERROR( lError, pMessage )                  \
+    do                                                            \
+    {                                                             \
+        if( lError < 0 ) {                                        \
+            LogError( pMessage " %s : %s.",                       \
+                      mbedtlsHighLevelCodeOrDefault( lError ),    \
+                      mbedtlsLowLevelCodeOrDefault( lError ) ); } \
     } while( 0 )
 
-#define MBEDTLS_LOG_IF_ERROR( lError, pFormatString, ... )  \
-    do                                                      \
-    {                                                       \
-        if( lError < 0 )                                    \
-        LogError( pFormatString " %s : %s.", __VA_ARGS__,   \
-                  mbedtlsHighLevelCodeOrDefault( lError ),  \
-                  mbedtlsLowLevelCodeOrDefault( lError ) ); \
+#define MBEDTLS_LOG_IF_ERROR( lError, pFormatString, ... )        \
+    do                                                            \
+    {                                                             \
+        if( lError < 0 ) {                                        \
+            LogError( pFormatString " %s : %s.", __VA_ARGS__,     \
+                      mbedtlsHighLevelCodeOrDefault( lError ),    \
+                      mbedtlsLowLevelCodeOrDefault( lError ) ); } \
```

Comment:
Uncrustify changed these lines. Rolling them into this PR since checks fail otherwise.